webapp/lib/view.dart

```diff
@@ -301,22 +300,38 @@ class ConfigurationViewTagResponsesPartial {
       ..classes.add('tag-responses__content');
   }
 
-  void renderResponses(String tag, Map<String, List<String>> responses) {
+  void renderResponses(String tag, List<String> responseLanguages, List<List<String>> responses) {
     tagResponsesElement.children.clear();
     _tagResponsesHeader.children.clear();
     _tagResponsesBody.children.clear();
-    responses.forEach((language, responseSet) {
-      _tagResponsesHeader.append(new HeadingElement.h5()..text = language);
-      var items = new DivElement()..classes.add('tag-responses__items');
-      for (int i = 0; i < responseSet.length; i++) {
-        var response = responseSet[i];
-        var item = new SpanElement()
+    responseLanguages.forEach((lang) => _tagResponsesHeader.append(
+      new HeadingElement.h5()
+        ..classes.add('tag-responses__header-title')
+        ..text = lang
+    ));
+    for (int i = 0; i < responses.length; i++) {
+      var item = new DivElement()
           ..classes.add('tag-responses__item-row')
           ..append(
+            new DivElement()
+            ..draggable = true
+            ..classes.add('tag-responses__item-drag-handle')
+            ..onDragStart.listen((event) {
+              var handle = event.target as Element;
+              var parent = handle.parent;
+              handle.classes.add('tag-responses__item-drag-handle_dragging');
+              var payload = {};
+              parent.children.skip(1).forEach((el) => payload[el.attributes['language']] = el.text);
```

Comment:
I think I'd prefer it if this was defined as a for loop rather than using `forEach`, to make it a bit clearer that the effect of this iteration is not on the `children` list, but on `payload` - as it is, it's a bit hard to interpret that this is what's happening

Reply:
Changed it to use a for..loop